serverless/service/__init__.py

```diff
@@ -82,16 +82,24 @@ def preset(self, **kwargs):
     def enable(self, feature):
         feature.enable(self)
 
-    def render(self, output=None):
+    def render(self, output=None, attach_file_auto_generated_warning=True):
```

Comment:
I think we could simplify this function slightly

```suggestion
    def render(self, output=None, auto_generated_warning=True):
        import __main__ as main

        output = output if output else open(Path(main.__file__).stem, "w+")
        if auto_generated_warning:
            output.write("# DO NOT edit this file directly, it was generated based on serverless.yml.py\n\n")

        output.write(str(self))
```